app/src/main/java/io/pumpkinz/pumpkinreader/model/Comment.java

```diff
@@ -0,0 +1,31 @@
+package io.pumpkinz.pumpkinreader.model;
+
+
+import java.io.Serializable;
+import java.util.Date;
+
+public class Comment implements Serializable {
```

Comment:
Oh to pass it to an intent.
From what I read in Effective Java, we should `implement Serializable judiciously` and we should add `private static final long serialVersionUID = randomLongValue`.

But if it's what Android suggest, I think it's OK for now, we can iterate later.